### ISSUE TITLE
Error on making a Dict

### ISSUE THREAD

**Issue by ranjanan** [JuliaStats/RCall.jl#85]
The following code:
```
using RCall
d = Dict(1=>2, 2=> 3)
RObject(d)
```
errors out with:
```
ERROR: UndefVarError: allocVector not defined
 in sexp at /Users/ranjan/.julia/v0.4/RCall/src/convert-base.jl:212
 in call at /Users/ranjan/.julia/v0.4/RCall/src/types.jl:270
```
Could you please point me to where `allocVector` is meant to be defined?

**Comment by randy3k** [comment]
It should be fixed in 4ba8123.

**Comment by ranjanan** [comment]
Yes it is.
